### PR TITLE
implements a way to win

### PR DESCRIPTION
**Pull request by muggell**

### CODE REVIEW

File: src/maga/character/Player.java [hunk 1]
```diff
@@ -148,5 +154,18 @@ public void useItem(Command command) {
             }
         }
     }
+    /**
+     * Returns the state of tweeted.
+     * @return true of false.
+     */
+    public boolean hasTweeted() {
+        return this.tweeted;
+        }
```
Comment (quartercastle) [inline src/maga/character/Player.java]: Identation! 😄

**Review by quartercastle — CHANGES_REQUESTED** [review]
👍

File: src/maga/Game.java [hunk 2]
```diff
@@ -252,4 +249,31 @@ private boolean youLose(){
...
+            Console.print(
+                "",
+                "Congratulations, you won the game!",
+                "",
+                "---------------------------------------",
+                "You made it in " + steps + " steps, in " + elapsedTime + " seconds!",
+                "You scored: " + finalScore,
+                "---------------------------------------"
+            );
+            return true;
```
Comment (quartercastle) [inline src/maga/Game.java]: Identation! 😄

File: src/maga/character/Player.java (hunk 1 above)
Comment (muggell) [reply src/maga/character/Player.java]: fixed d5128e42007fd662392b9f75fb85e68f9ef52486

**Review by quartercastle — CHANGES_REQUESTED** [review]

File: src/maga/Game.java (hunk 2 above)
Comment (muggell) [reply src/maga/Game.java]: fixed cb8ad2feb7ad68271c805855663df9cdf3decf5f

**Review by quartercastle — APPROVED** [review]
👍

**Pull request merged**